src/controller/Controller.java

```diff
@@ -3,30 +3,72 @@
 import view.View;
 import model.*;
 
+import java.io.IOException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.logging.*;
+
 public class Controller {
 	
 	private View view;
-	private Password assignedPassword;
-	private Password enteredPassword;
+	private Password assignedPassword; // password system gives them
+	private Password enteredPassword; // what they give us
 	
 	public Controller(){
 		//TODO: randomly assign a password here
+		assignedPassword = new Password(/*take in assigned values */ );
+		
+		enteredPassword = new Password();
 		view = new View(this);
 		view.setVisible(true);
-		
 	}
 	
 	public void handleSubmit(){
-		//TODO: make this do things
+		boolean success = assignedPassword == enteredPassword;
+		
+		// TODO: change the state based off of success
+		
+		String event = success ? "success" : "failure";
+		// log success or failure event
+		logEvent("user,scheme,login," + event);	
```

Comment:
yeah, since we haven't figured out how we're keeping track of that i've just put that in as a placeholder for now.